### PR TITLE
New port: DoomRL

### PR DESCRIPTION
**Pull request by ohol-vitaliy**
# New Port for DoomRL

## Game Information
- **Title**: DoomRL
- **URL**: https://github.com/ohol-vitaliy/doomrl

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
```
doomrl
├── doomrl
│   ├── backup
│   ├── colors.lua
│   ├── config.lua
│   ├── core.wad
│   ├── drl
│   ├── drl.gptk
│   ├── drl.gptk2
│   ├── drl.wad
│   ├── libs
│   ├── licenses
│   ├── manual.txt
│   ├── modules
│   ├── mortem
│   ├── mp3
│   ├── musichq.lua
│   ├── screenshot
│   ├── settings.lua
│   ├── soundhq.lua
│   ├── version_api.txt
│   ├── version.txt
│   └── wavhq
├── DoomRL.sh
├── gameinfo.xml
├── port.json
├── README.md
└── screenshot.png
```

### CODE REVIEW

File: ports/doomrl/DoomRL.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+source $controlfolder/control.txt
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+get_controls
+
+GAMEDIR="/$directory/ports/doomrl"
+BINARY="drl"
+
+export DEVICE_ARCH="${DEVICE_ARCH:-aarch64}"
+export LD_LIBRARY_PATH="$GAMEDIR/libs:/usr/lib/:/usr/lib/aarch64-linux-gnu/:/usr/lib32/:$LD_LIBRARY_PATH"
```
Comment (Cebion) [inline ports/doomrl/DoomRL.sh:24]: Wrong LD Library Export. We generally don't need to export /usr/lib  and lib32 is for armhf which this also isn't.
```suggestion
export LD_LIBRARY_PATH="$GAMEDIR/libs.${DEVICE_ARCH}:$LD_LIBRARY_PATH"
```

File: ports/doomrl/DoomRL.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+export XDG_CONFIG_HOME="$GAMEDIR"
```
Comment (Cebion) [inline ports/doomrl/DoomRL.sh:25]: Only xdg_data_home is used.
```suggestion
```

File: ports/doomrl/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Here are the steps you need to take:
+- Go to the [DoomRL downloads page](https://drl.chaosforge.org/downloads)
+- Download Windows or Linux archive 
+- Copy `mp3` directory from archive into `ports/doomrl/`
+
+## Version
+0.9.9.8
+
+## Compile
+Follow steps from this repo:
```
Comment (Cebion) [inline ports/doomrl/README.md:42]:
```suggestion
```

File: ports/doomrl/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+https://github.com/ohol-vitaliy/PortMaster-DoomRL
```
Comment (Cebion) [inline ports/doomrl/README.md:43]:
```suggestion
```
git clone https://github.com/ohol-vitaliy/PortMaster-DoomRL.git
docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
docker build . --platform linux/arm64 --rm -t drl_build
docker run --rm drl_build cat /root/drl.zip > drl.zip
```
```

**Review by cdeletre — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**